inference-engine/src/mkldnn_plugin/mkldnn_node.h

```diff
@@ -483,20 +496,40 @@ class MKLDNNNode : public InferenceEngine::details::no_copy {
         return desc.outputNumbers();
     }
 
+    const ITTCounters & perfCounters() const {
+        return profiling;
+    }
+
+    ITTCounters & perfCounters() {
+        return profiling;
+    }
+
     template<typename To>
     class Register {
     public:
         explicit Register(const std::string& type) {
             MKLDNNNode::AddNode(type,
-                    [](const InferenceEngine::CNNLayerPtr& layer, const mkldnn::engine& eng,
+                    [type](const InferenceEngine::CNNLayerPtr& layer, const mkldnn::engine& eng,
                             MKLDNNWeightsSharing::Ptr &w_cache)
                     -> MKLDNNNode* {
-                        return new To(layer, eng, w_cache);
+                        MKLDNNNode *node = new To(layer, eng, w_cache);
+
+                        auto const &name = type;
```

Comment:
All perf counters are parametrized via type name. Excluding only one which has node name. I see some inconsistency here.

Reply:
Everything is all right here. This performance counter is created unique for each node.